contracts/ChainlinkPriceFeedV3.sol

```diff
@@ -42,14 +42,15 @@ contract ChainlinkPriceFeedV3 is IChainlinkPriceFeedV3, IPriceFeedUpdate, BlockC
     }
 
     /// @notice anyone can help with updating
-    /// @dev keep this function for PriceFeedUpdater for updating, since multiple updates
-    ///      with the same timestamp will get reverted in CumulativeTwap._update()
+    /// @dev this function is used by PriceFeedUpdater for _lastValidPrice and twap updating,
+    ///      The keeper can invoke callstatic on this function to check if _lastValidPrice needs to be updated.
+    ///      If _lastValidPrice is updated, then update the observation array for twap calculating.
```

Comment:
It also invovles _lastValidTimestamp


```suggestion
    /// @dev this function is used by PriceFeedUpdater for updating observation array.
    ///      The keeper can invoke callstatic on this function to check if _lastValidPrice, _lastValidTimestamp or observation arry needs to be updated.
```

Reply:
fixed in [7ebde82](https://github.com/perpetual-protocol/perp-oracle-contract/pull/79/commits/7ebde82798907cdc0a5835bcac5444952e1efe5b)